docs/data-model.md

```diff
@@ -16,13 +16,32 @@ There are a few things to bear in mind when reading this:
 
 ```mermaid
 erDiagram
-  Placement {
+  Subject {
+    uuid id PK
+    string subject_area "primary or secondary"
+    string name
+    string code "code comes from publish API, but it is not a FK"
```

Comment:
"but it is not a FK" – is this true?

I think the API endpoint returns data in an odd format. But my understanding is that the subject "code" is the unique identifier for a subject.

```suggestion
    string code FK "Primary key for subjects in the Teacher Training Courses API"
```

Reply:
The trouble with that is that code can be null (the Modern languages subject has a code of null) so I think calling it a FK is misleading.

Reply:
Fair point. Marking as resolved.